app/src/main/java/com/codesoom/assignment/DemoHttpHandler.java

```diff
@@ -0,0 +1,118 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.models.Task;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.*;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class DemoHttpHandler implements HttpHandler {
+    private ObjectMapper objectMapper = new ObjectMapper();
+
+
+    private List<Task> tasks = new ArrayList<>();
+
+//    public DemoHttpHandler() {
+//        Task task = new Task();
+//        task.setId(1L);
+//        task.setTitle("DO nothing...");
+//
+//        tasks.add(task);
+//
+//    }
+
+
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+        // REST - CURD
+        // 1. Method - Get, POST, PUT/PATCH, DELETE, ...
+        // 2. Path - "/", "/tasks", "/tasks/1", ...
+        // 3. Headers, Body(Content)
+
+        String method = exchange.getRequestMethod();
+        URI uri = exchange.getRequestURI();
+        String path = uri.getPath();
```

Comment:
변경하지 않을 값을 선언하게 될 경우엔 `final`을 붙여 보면 어떨까요?
```suggestion
        final String method = exchange.getRequestMethod();
        final URI uri = exchange.getRequestURI();
        final String path = uri.getPath();
```